plugins/modules/template_workflow_manager.py

```diff
@@ -1484,7 +1501,7 @@ def get_containing_templates(self, containing_templates):
 
             language = item.get("language")
             if language is None:
-                self.msg = "language is mandatory under containing templates"
+                self.msg = "language is required under containing templates"
```

Comment:
The parameter 'language' is required under 'containing_templates' but not provided.